core/src/main/java/com/netease/arctic/table/blocker/RenewableBlocker.java

```diff
@@ -93,10 +95,16 @@ public void renewAsync() {
 
   private void doRenew() {
     try {
+      Set<String> validBlockers = amsClient.getBlockers(tableIdentifier.buildTableIdentifier())
```

Comment:
Why this change is needed?
The new implementation require two rpc each time when renew, as the old implementation looks fine to me.

Reply:
The old implementation brings annoying error logs both in the spark client and AMS. I can change the log level to avoid part of these error logs.

Reply:
I found all these annoying errors could be avoided by using appropriate log level, I will change the code.